### ISSUE TITLE
[Bug] Cannot use ebpfspy in Debian

### ISSUE THREAD

**Issue by et84121** [pyroscope-io/pyroscope#114]
I use pyroscope with ebpfspy in [Python docker official Images](https://hub.docker.com/_/python/) and it's based on Debian Buster image.

According to this [article](https://blog.lazy-evaluation.net/posts/linux/bpf-superpowers-for-linux.html) 
>On Debian the package maintainer has decided to name all the tools with an -bpfcc suffix not to collide with tools from the perf suite

there is no directory in `/usr/share/bcc/tools/profile` ,  so Pyroscope with  [ebpfspy](https://github.com/pyroscope-io/pyroscope/blob/28f31b86718751f6668f184849527e0ef5c30ec2/pkg/agent/ebpfspy/session.go#L45) would get error message as following
```
error when starting session: "Could not find profile.py at '/usr/share/bcc/tools/profile'. Visit https://github.com/iovisor/bcc/blob/master/INSTALL.md for instructions on how to install it""
```
I think this is a bug and should be fixed

**Comment by petethepig** [comment]
Thanks for reporting this. We'll fix it in the next release

**Comment by petethepig** [comment]
@et84121 This has been fixed in [0.0.28](https://github.com/pyroscope-io/pyroscope/blob/main/CHANGELOG.md#0028-2021-03-11)

**Comment by et84121** [comment]
thanks you for quickly bug fix
